libraries/rush-lib/src/logic/installManager/WorkspaceInstallManager.ts

```diff
@@ -648,18 +648,17 @@ export class WorkspaceInstallManager extends BaseInstallManager {
         for (const value of Object.values(hoistedDependencies)) {
           for (const [filePath, type] of Object.entries(value)) {
             if (type === 'public') {
-              // If we don't already have a symlink for this package, create one
-              if (!FileSystem.exists(`${projectNodeModulesPath}/${filePath}`)) {
-                const parentDir: string = Utilities.trimAfterLastSlash(
-                  `${projectNodeModulesPath}/${filePath}`
-                );
-                await FileSystem.ensureFolderAsync(parentDir);
-                BaseLinkManager._createSymlink({
-                  linkTargetPath: `${tempNodeModulesPath}/${filePath}`,
-                  newLinkPath: `${projectNodeModulesPath}/${filePath}`,
-                  symlinkKind: SymlinkKind.Directory
-                });
+              if (BaseLinkManager._symlinkExists(`${projectNodeModulesPath}/${filePath}`)) {
+                await FileSystem.deleteFolderAsync(`${projectNodeModulesPath}/${filePath}`);
```

Comment:
Note that if `` `${projectNodeModulesPath}/${filePath}` `` is a symlink, then `deleteFolderAsync()` will delete the symlink not the folder pointed to by the symlink.

Reply:
Yes this is the intended behaviour